xml/System.Net.Sockets/SocketType.xml

```diff
@@ -66,7 +66,7 @@
         <ReturnType>System.Net.Sockets.SocketType</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Supports datagrams, which are connectionless, unreliable messages of a fixed (typically small) maximum length. Messages might be lost or duplicated and might arrive out of order. A <see cref="T:System.Net.Sockets.Socket" /> of type <see cref="F:System.Net.Sockets.SocketType.Dgram" /> requires no connection prior to sending and receiving data, and can communicate with multiple peers. <see cref="F:System.Net.Sockets.SocketType.Dgram" /> uses the Datagram Protocol (<see cref="F:System.Net.Sockets.ProtocolType.Udp" />) and the <see cref="F:System.Net.Sockets.AddressFamily.InterNetwork" /><see cref="T:System.Net.Sockets.AddressFamily" />.</summary>
+        <summary>Supports datagrams, which are connectionless, unreliable messages of a fixed (typically small) maximum length. Messages might be lost or duplicated and might arrive out of order. A <see cref="T:System.Net.Sockets.Socket" /> of type <see cref="F:System.Net.Sockets.SocketType.Dgram" /> requires no connection prior to sending and receiving data, and can communicate with multiple peers. <see cref="F:System.Net.Sockets.SocketType.Dgram" /> uses the Datagram Protocol (<see cref="F:System.Net.Sockets.ProtocolType.Udp" />) and the <see cref="F:System.Net.Sockets.AddressFamily.InterNetwork" />.<see cref="T:System.Net.Sockets.AddressFamily" />.</summary>
```

Comment:
@rpetrusha 

I thought that this was trying to say (in simple markdown):

`` `SocketType.Dgram` uses the Datagram Protocol (`ProtocolType.Udp`) and the `InterNetwork` `AddressFamily`.``

If it was supposed to be a type/member name, then I think the definite article would not be there:

`` `SocketType.Dgram` uses the Datagram Protocol (`ProtocolType.Udp`) and `AddressFamily.InterNetwork`.``

Also, the space is there in [an older version of this doc on MSDN](https://msdn.microsoft.com/en-us/library/system.net.sockets.sockettype(v=vs.100).aspx).

Though I'm not a native speaker and I'm arguing about what it was supposed to be when first written, not about what it should be now, so I'm fine with your suggestion.